go.mod

```diff
@@ -4,21 +4,26 @@ go 1.14
 
 require (
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
-	github.com/google/go-cmp v0.4.0
+	github.com/google/go-cmp v0.5.2
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/openshift/api v0.0.0-20201130121019-19e3831bc513
 	github.com/openshift/build-machinery-go v0.0.0-20200917070002-f171684f77ab
 	github.com/openshift/client-go v0.0.0-20200827190008-3062137373b5
 	github.com/openshift/library-go v0.0.0-20200909144351-f29d76719396
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus-operator/prometheus-operator v0.44.1
+	github.com/prometheus/client_golang v1.8.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.0.0-20200813134508-3edf25e44fcc // indirect
 	k8s.io/api v0.19.2
-	k8s.io/apiextensions-apiserver v0.19.0
+	k8s.io/apiextensions-apiserver v0.19.2
 	k8s.io/apimachinery v0.19.2
-	k8s.io/client-go v0.19.0
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
This does not actually result in downgrade of client-go(check `go.sum`), but used client-go is still `v0.19.2` but what is happening is - one of the dependencies of prometheus-operator depends on `v0.12.0` - https://github.com/prometheus-operator/prometheus-operator/blob/master/go.mod#L48 and because of golang-1.13 semver check it is almost impossible to use client-go `v0.12.0` without using replace directive.

I have not manually added this line - but `go mod vendor` and `go mod tidy` will replace `v.0.19.2` with this version.